src/keytransform.js

```diff
@@ -2,13 +2,27 @@
 
 const { Adapter, utils } = require('interface-datastore')
 const map = utils.map
+/**
+ * @typedef {import('interface-datastore/src/types').Datastore}Datastore
```

Comment:
```suggestion
 * @typedef {import('interface-datastore/src/types').Datastore} Datastore
```

Reply:
@hugomrdias shouldn't the linter pick this up?

Reply:
Apparently no rule for this